context_response_test.go

```diff
@@ -701,3 +702,60 @@ func TestResponse_Status(t *testing.T) {
 		})
 	}
 }
+
+func TestResponse_Stream(t *testing.T) {
+	var (
+		err        error
+		fiber      *Route
+		req        *http.Request
+		mockConfig *configmocks.Config
+	)
+
+	beforeEach := func() {
+		mockConfig = &configmocks.Config{}
+		mockConfig.EXPECT().GetBool("http.drivers.fiber.prefork", false).Return(false).Once()
+		mockConfig.EXPECT().GetInt("http.drivers.fiber.body_limit", 4096).Return(4096).Once()
+		mockConfig.EXPECT().GetInt("http.drivers.fiber.header_limit", 4096).Return(4096).Once()
+		ConfigFacade = mockConfig
+	}
+
+	t.Run("Stream", func(t *testing.T) {
```

Comment:
Actually, we don't need this line and `beforeEach` in this case anymore, you can write the single test case directly.